pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Create AsyncAPI Document for applications consuming from Kafka
+description: Explore configuring AsyncAPI document for Kafka messages.
+weight: 20
+---
+
```

Comment:
just an observation, something you could consult with other writers

tutorials set called Getting Started have a dedicated short `Introduction` document that is placed in `index.md` as a main document for getting started, an intro (referring to it as you also place first tutorial in `index.md`)

since there are going to be 4 kafka tutorials, maybe it would make sense to have a separate introduction to these? with generic background and short explanation about the flow, what given tutorials is about and the suggested order of reading?

anyway, just an idea you could explore and discuss with others

Reply:
Sounds good, I'll discuss it with others 👍

Reply:
Not for this case, because this is not a Quickstart. We wrote a 4 part tutorial series, but they do not all share the same introduction. 

We have been using the "Introduction" in all tutorials as the appropriate place to introduce your readers to the goal is that specific tutorial, so let's maintain consistency. 😄